model_compression_toolkit/exporter/model_wrapper/pytorch/validate_layer.py

```diff
@@ -33,10 +35,10 @@ def is_pytorch_layer_exportable(layer: Any) -> bool:
         """
         if isinstance(layer, PytorchQuantizationWrapper):
             quantizers = list(layer.weights_quantizers.values())
-            quantizers.extend(layer.activation_quantizers)
-            if all([isinstance(q, BasePyTorchInferableQuantizer) for q in quantizers]):
-                return True
-        return False
+            return all([isinstance(q, BasePyTorchInferableQuantizer) for q in quantizers])
+        elif isinstance(layer, PytorchActivationQuantizationHolder):
+            return isinstance(layer.activation_holder_quantizer, BasePyTorchInferableQuantizer)
```

Comment:
Add check that every layer must be an nn.module.
Maybe it's better to raise an error with an informative error message in cases it's not a valid layer?